frontend/language/src/nb.json

```diff
@@ -1237,6 +1237,11 @@
   "ux_editor.component_properties.optionsId": "Kodeliste",
   "ux_editor.component_properties.overrides": "Overstyringer",
   "ux_editor.component_properties.overrides_description": "Overstyringer per komponent for oppsummeringen",
+  "ux_editor.component_properties.overrides_is_compact": "Kompakt visning",
+  "ux_editor.component_properties.overrides_list": "Liste",
+  "ux_editor.component_properties.overrides_not_set": "Ikke satt",
+  "ux_editor.component_properties.overrides_string": "String",
```

Comment:
Just checking that we are not going to use `Text` here instead?

Reply:
@framitdavid, can you confirm this? 😄

Reply:
`String` er et engelsk ord og burde derfor ikke brukes i den norske versjonen. Jeg stemmer for `Tekst` eller `Streng`. Jeg tror `Tekst` kanskje er lettest for våre brukere å forstå? 🤔

Reply:
Enig med Tekst 😄